docs/1_introduction.md

```diff
@@ -0,0 +1,43 @@
+# 💉 Introduction to the [ReVanced patcher](https://github.com/revanced/revanced-patcher)
+
+Familiarize yourself with the [ReVanced patcher](https://github.com/revanced/revanced-patcher).
+
+## 📙 How it works
+
+```kt
+// Prepare patches to apply and files to merge
+
+val patches = PatchBundle.Jar("revanced-patches.jar").loadPatches()
+val mergeList = listOf("integrations.apk")
+
+// Create the options for the patcher
+
+val options = PatcherOptions(
+     inputFile = File("some.apk"),
+     resourceCacheDirectory = File("cache"),
+)
+
+// Create the patcher and add the prepared patches and files
+
+val patcher = Patcher(options)
+    .also { it.addPatches(patches) }
+    .also { it.addFiles(mergeList) }
+
+// Execute and save the patched files
+
+patcher.executePatches().forEach { (patch, result) ->
+    val log = if (!result.isSuccess)
+        "failed"
+    else
+        "succeeded"
+    println("$patch $log")
+}
+
+val result = patcher.save()
```

Comment:
Missing docs for the result type, either explain it here or redirect the reader to a place where they can understand the result type (e.g, javadocs/kdoc).

Reply:
This docs do not go into detail on the patcher. Instead they should give the necessary understanding for how the patches are used by the patcher.

Reply:
Yes, something similar like this is planned, but docs for the patcher are not available yet, hence no link exists, if you want you can prepare a template and link to it.